web/src/components/RemoteContentSandBox.ts

```diff
@@ -171,6 +174,8 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     }
 
     img {
+      filter: ${props => props.theme.isContrastTheme && 'invert(1)'};
```

Comment:
❓ Same as above.

Reply:
we will leave it as it is right now I guess to not block the PR?